source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -33,6 +33,11 @@ The following attributes were added :
 `enum` tag was added to following attributes:
 - NIDCPOWER_ATTRIBUTE_VOLTAGE_LEVEL_AUTORANGE
 - NIDCPOWER_ATTRIBUTE_POWER_LINE_FREQUENCY
+- NIDCPOWER_ATTR_CURRENT_LIMIT_BEHAVIOR
+- NIDCPOWER_ATTR_CURRENT_LIMIT_AUTORANGE
+- NIDCPOWER_ATTR_CURRENT_LEVEL_AUTORANGE
+- NIDCPOWER_ATTR_VOLTAGE_LIMIT_AUTORANGE
+- NIDCPOWER_ATTR_AUTO_ZERO
```

Comment:
The attribute you have modified is "AUTORANGE" not "AUTO_ZERO"

Reply:
Ohh yes.. Copy-pasted wrong name. I have changed it.